src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -11,13 +12,23 @@ public class LEDSubsystem extends SubsystemBase {
 
     private final Timer blinkTimer;
     private static final double BLINK_DURATION_SECONDS = 0.5;
+    private final double APRIL_BLINK_DURATION = .1;
     private static final Color BLINK_COLOR = new Color(0, 0, 0);
     private boolean blinking = false;
+    public boolean manual = false; //when driver is directly controlling leds
 
     private static final double BRIGHTNESS_SCALE_FACTOR = 0.25;
+    private static final double INPUT_DEADZONE = .1;
 
     private Color color = new Color(192, 8, 254);
```

Comment:
what is this used for?

Reply:
ah, what if this was like a manualOverrideColor, and when it's null, it's not used and we do the default logic? that allows us to minimize states